Add dns-query for code review

srsudar opened the pull request.

srsudar committed:
Add dns-query for code review

mdwelsh commented on chromeapp/test/scripts/dnssd/dns-query-test.js:
```diff
@@ -0,0 +1,34 @@
+  var name = 'www.example.com';
+  var queryType = 3;
+  var queryClass = 4;
+  // // Corresponds to 155.33.17.68
```
Dead code?

srsudar replied:
Whoops; removed.

Done.

mdwelsh commented on chromeapp/test/scripts/dnssd/dns-query-test.js:
```diff
@@ -0,0 +1,34 @@
+
+  var serialized = expected.serialize();
+
+  var actual = dnsQuery.createQueryFromByteArray(serialized);
```
Add test where startByte != 0

srsudar replied:
done

srsudar committed:
Remove dead code and add test for startbyte != 0

Some dead code was hanging around in dns-query-test; kill it.
Also add a test to ensure that deserializing a DnsQuery from
a ByteArray respects the given byte offset into this array.
